subsys/shell/Kconfig

```diff
@@ -33,3 +33,89 @@ config CONSOLE_SHELL_MAX_CMD_QUEUED
 source "subsys/shell/modules/Kconfig"
 
 endif
+
+config SHELL
```

Comment:
Just tried this with `make menuconfig` and the shell support looks really bad as all the config options are in the "root". The Kconfig seems to be missing `menu "Shell support"` or similar command.

Reply:
Good point, I will update it.

Reply:
@jukkar : done